forge-gui/res/cardsfolder/upcoming/highway_robbery.txt

```diff
@@ -0,0 +1,9 @@
+Name:Highway Robbery
+ManaCost:1 R
+Types:Sorcery
+A:SP$ GenericChoice | Choices$ DBDiscardToDraw,DBSacToDraw | SpellDescription$ You may discard a card or sacrifice a land. If you do, draw two cards.
```

Comment:
Should there also be a "do neither" option? Since this is all kind of optional?

Or in that fairly strange case should we just let people figure out that they can pick one and then hit cancel

Reply:
Hmm I know this is kinda janky, but the only similar card i've found was Crypt Lurker which is scripted similar. 
And ofc you don't need to pay the unlesscost but it can be confusing for players